build_tools/cmake/iree_macros.cmake

```diff
@@ -514,6 +514,37 @@ function(iree_symlink_tool)
   )
 endfunction()
 
+#-------------------------------------------------------------------------------
+# Test timeout values
+#-------------------------------------------------------------------------------
+
+# iree_get_timeout_seconds()
+#
+# Evaluates a Bazel-like timeout string to a number of seconds.
+# Allowed timeout values:  "short", "moderate", "long".
+# Default is "short".
+#
+# This is aligned with our `native_test` function in Bazel. This differs from
```

Comment:
I think we should implement this in all our "leaf" test macros, e.g. `cc_test` as well